CODEOWNERS

```diff
@@ -266,3 +266,9 @@ k8s/**/cluster-*/pcq/pcq-consolidation-service.yaml @hmcts/pcq-admins
 ### RSE
 k8s/demo/common/rse-istio-poc/** @hmcts/rse
 k8s/aat/common/rse/check.yaml @hmcts/rse
+
+### UCMC
+k8s/demo/common/ucmc/namespace.yaml @hmcts/ucmc-admins
```

Comment:
you can leave namespace as ** but not a blocker

Reply:
Good point, we'll probably be adding more files there soon. Changed